docs/sections/getting_started.md

```diff
@@ -128,17 +128,25 @@ popper run
 
 You should see the output of actions printed to the terminal.
 
+To obtain more detailed information of what this command does, you can pass the `--help` flag to it:
+
+```bash
+popper run --help
+```
+
+**Note:** All Popper subcommands allow you to pass `--help` flag to it to get more information about what the command does.
```

Comment:
let's make this follow the same format as the other notes in this doc.

Reply:
I'd get right on this.

Reply:
I didn't see there was a styling format. Wow, open-source is fun in some ways